tests/func/test_catalog.py

```diff
@@ -971,42 +971,6 @@ def test_query_subprocess_wrong_return_code(mock_popen, cloud_test_catalog):
         assert str(exc_info.value).startswith("Query script exited with error code 1")
 
 
-def test_query_last_statement_not_expression(mock_popen, cloud_test_catalog):
```

Comment:
These tests are mocked tests. And since these tests are no longer a failure, they are not useful.